### ISSUE TITLE
Hyperparameter ``max_grad_norm`` not utilized in RL Algorithms

### ISSUE THREAD

**Issue by nobodyPerfecZ** [keraJLi/rejax#16]
The current implementation and configurations include a hyperparameter called ``max_grad_norm`` (inside of ``algorithm.py``). However, it appears that this hyperparameter is not being utilized in any of the RL algorithms.

**Comment by keraJLi** [comment]
Hi @nobodyPerfecZ! Thank you for raising the issue. I've fixed it on the main branch. Feel free to install from source via `pip install git+https://github.com/keraJLi/rejax` to use gradient clipping.
